app/boards/shields/helix/helix.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Enables Sleep functionality (Enabled by default, comment to disable.)
+CONFIG_ZMK_SLEEP=y
```

Comment:
Eventually ZMK sleep stuff should be enabled by default, so you can remove this. For now each person that wants it enabled should do so manually.